Formula/root.rb

```diff
@@ -52,10 +48,18 @@ class Root < Formula
   depends_on "xz" # for LZMA
   depends_on "zstd"
 
+  uses_from_macos "libxml2"
+
   conflicts_with "glew", because: "root ships its own copy of glew"
 
   skip_clean "bin"
 
+  # Can be removed post 6.22.08
+  patch :p0 do
```

Comment:
```suggestion
  patch do
```
Strip level is off. You need to strip the `a/` and `b/` from the paths in the patch.

Reply:
Missed that, thanks!